org.kde.krename.json

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "org.kde.krename",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "krename",
+    "rename-icon": "krename",
+    "finish-args": ["--share=ipc", "--socket=fallback-x11", "--socket=wayland", "--filesystem=host" ],
+
+    "modules": [
+        {
+            "name": "libexif",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2",
+                    "sha256": "d47564c433b733d83b6704c70477e0a4067811d184ec565258ac563d8223f6ae"
+                }
+            ]
+        },
+        {
+            "name": "taglib",
+            "config-opts": [ "-DBUILD_SHARED_LIBS=ON" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.github.io/releases/taglib-1.12.tar.gz",
+                    "sha256": "7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703"
+                }
+            ]
+        },
+        {
+            "name": "kjs",
+            "config-opts": [ "-DENABLE_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/Attic/frameworks/5.83/portingAids/kjs-5.83.0.tar.xz",
```

Comment:
The latest version is at https://download.kde.org/stable/frameworks/5.89/portingAids/